src/components/CoreNodeName.vue

```diff
@@ -31,31 +30,20 @@ export default {
     ])
   },
   props: [
-    'node-hash',
-    'node-type'
+    'node-hash'
   ],
   data () {
     return {
-      node: null
+      node: undefined
```

Comment:
Can you explain why this switch from `null` to `undefined` ?

Reply:
Default behaviour of [Array.find](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/find) is to return undefined if there is no match